start

```diff
@@ -13,10 +13,22 @@
 
 import subprocess
 from pathlib import Path
+from typing import Iterator
 
 PWD = Path(__file__).parent
 
 
+def translation_volume_mounts() -> Iterator[str]:
+    """Return the CLI args to mount each language in the translations/ folder.
+
+    Unlike the root `Dockerfile`, we cannot use `-v $PWD/translations:/home/node/app/docs` because
+    Docker complains that the volume is already mounted when we mount the `/docs` folder. So, instead
+    we need to be more specific to mount each language's folder.
+    """
+    for folder in PWD.glob("translations/*"):
+        yield from ["-v", f"{folder}:/home/node/app/docs/{folder.name}"]
```

Comment:
This `yield from` is a nifty Python feature. This function is a _generator_, meaning it only returns the next element once we iterate to it. That is rather than storing everything all at once in memory. This is also how `PWD.glob()` works.

Here, I mostly use the iterator for concise code. Rather than having a `result: list[str]` variable etc.